Scripts/build/Jenkins/Jenkinsfile

```diff
@@ -498,12 +497,12 @@ def UploadAPLogs(Map options, String branchName, String platformName, String job
             }
             def command = "${pythonPath} -u ${s3UploadScriptPath} --base_dir ${apLogsPath} " +
                           "--file_regex \".*\" --bucket ${env.AP_LOGS_S3_BUCKET} " +
-                          "--search_subdirectories True --key_prefix ${env.JENKINS_JOB_NAME}/${branchName}/${env.BUILD_NUMBER}/${platformName}/${jobName} " +
+                          "--search_subdirectories True --key_prefix ${PROJECT_REPOSITORY_NAME}/${branchName}/${env.BUILD_NUMBER}/${platformName}/${jobName} " +
```

Comment:
I don't see where PROJECT_REPOSITORY_NAME is set

Reply:
It's a const at the top of the file: https://github.com/o3de/o3de-atom-sampleviewer/blob/c865eaf5eead9061f2b6e188383c5dd824110732/Scripts/build/Jenkins/Jenkinsfile#L16

Reply:
I put it in place to ensure that the project job doesn't pollute the same space with the o3de jobs, since the main branches are all named the same

Reply:
Ok, env.JENKINS_JOB_NAME is the name of the Jenkins job, since ASV job name is different than o3de jobs, it won't pollute the o3de folder, you can see ASV artifacts are in o3de-atom-sampleviewer folder.
The problem of using PROJECT_REPOSITORY_NAME is that if we have two ASV Jenkins jobs, these two jobs will upload artifacts to the same place.

Reply:
Actually, the current artifacts for o3de-atom-sampleviewer folder are there because I tested this with this code.

Reply:
Hmmm... Did you run with env.JENKINS_JOB_NAME too? I cannot any ASV AP logs except "o3de-atom-sampleviewer" on S3...

Reply:
I did, the result was `null`. I'll give it a run with the old parameter so we can validate this